Additional info about creating new custom schema (#10550)
supabase/supabase: apps/docs/docs/reference/javascript/v1/initializing.mdx

```

By default the API server points to the `public` schema. You can enable other database schemas within the Dashboard.

> Go to `Settings > API > Schema` and add the schema which you want to expose to the API. You also need to grant `USAGE`on your new schema and your desire grant such as `SELECT, INSERT, UPDATE, DELETE` on objects those will be created in your new schema. 

Reviewer: ```suggestion
Go to `Settings > API > Schema` and add the schema which you want to expose to the API. You also need to grant `USAGE` on your new schema with the grants you desire, such as `SELECT, INSERT, UPDATE, DELETE`. 
```
